sovrin_client/agent/walleted_agent.py

```diff
@@ -123,8 +123,8 @@ def _saveWallet(self, wallet: Wallet, contextDir, walletName=None):
             fileName = normalizedWalletFileName(walletName)
             walletFilePath = self.walletSaver.saveWallet(
                 wallet, getWalletFilePath(contextDir, fileName))
-            self.logger.info('Active wallet "{}" saved ({})'.
-                             format(walletName, walletFilePath))
+            self.logger.debug('Active wallet "{}" saved ({})'.
```

Comment:
Why isn't it INFO? Do we have INFO level messages for Agents?
The current agents are rather test/demo one, so it makes sense to track some information (present in getting started?) in INFO level